integration-tests/environment.py

```diff
@@ -27,16 +27,6 @@ def before_all(context):
     context.cloudformation = boto3.resource("cloudformation")
     context.client = boto3.client("cloudformation")
 
-    config_path = os.path.join(context.sceptre_dir, "config", "9/B" + ".yaml")
-
-    with open(config_path, "r") as file:
-        file_data = file.read()
-
-    file_data = file_data.replace("{project_code}", context.project_code)
-
-    with open(config_path, "w") as file:
-        file.write(file_data)
-
```

Comment:
This was able to be completely removed because I'm not using Jinja syntax to reference the project_code properly. We should have done that all along... in any case, this chunk of integration tests code has been a thorn in the side of anyone running integration tests locally, since it actually produced changes in the files that couldn't be committed. With this gone, we're in much better state now.